### PR TITLE
[1LP][RFR] Automated BZ 1742839 applicable for CF5.11

### PR DESCRIPTION
**Pull request by sbulage**
## Purpose or Intent

Automated Ansible tests which stuck in queue and never finishes if "Provider Operations" role is disabled.
### PRT Run

{{pytest: cfme/tests/ansible/test_embedded_ansible_services.py -k "test_ansible_service_with_operations_role_disabled" --long-running -svvvv }}

### CODE REVIEW

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 1]
```diff
@@ -892,3 +892,31 @@ def _revert():
         if appliance.version < "5.11"
         else "Finished"
     )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1742839], server_roles=["-ems_operations"])
+def test_ansible_service_with_operations_role_disabled(appliance, ansible_catalog_item,
+ansible_service_catalog, ansible_service_funcscope, ansible_service_request_funcscope):
```
Comment (valaparthvi) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: **Optional -**
Can you format this?
```suggestion
def test_ansible_service_with_operations_role_disabled(
    appliance,
    ansible_catalog_item,
    ansible_service_catalog,
    ansible_service_funcscope,
    ansible_service_request_funcscope,
):
```

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 2]
```diff
@@ -892,3 +892,31 @@ def _revert():
...
+        initialEstimate: 1/3h
+        tags: ansible_embed
+    """
+    service_request = ansible_service_catalog.order()
+    service_request.wait_for_request(num_sec=300, delay=20)
+
+    view = navigate_to(ansible_service_funcscope, "Details")
+    assert (
+        view.provisioning.results.get_text_of("Status") == "successful"
+        if appliance.version < "5.11"
```
Comment (valaparthvi) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: Since this test is anyway ignored for 5.10, I don't think you need this.

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_services.py]: Nice Catch!!! Thanks @valaparthvi for pointing it out.

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_services.py]: Updated. @valaparthvi :smile:

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 3]
```diff
@@ -892,3 +892,27 @@ def _revert():
...
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/3h
+        tags: ansible_embed
+    """
+    service_request = ansible_service_catalog.order()
+    service_request.wait_for_request(num_sec=300, delay=20)
+
+    view = navigate_to(ansible_service_funcscope, "Details")
+    assert view.provisioning.results.get_text_of("Status") == "Finished"
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: @sbulage please add `status` as property of Service class. which will help to other as well.

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 4]
```diff
@@ -892,3 +892,27 @@ def _revert():
         if appliance.version < "5.11"
         else "Finished"
     )
+
+
+@pytest.mark.tier(3)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1742839], server_roles=["-ems_operations"])
```
Comment (valaparthvi) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: TIL : `-ems_operations` this is new! :champagne:

File: cfme/tests/ansible/test_embedded_ansible_services.py (hunk 3 above)
Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_services.py]: Sure @digitronik I will also use it into #9076

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_services.py]: @digitronik I have updated.

**Pull request merged**